src/faq/plugin.ts

```diff
@@ -31,301 +29,330 @@ const faqExtension: JupyterLabPlugin<void> = {
+  private _subHeadings: string[];
+  private _basicsQuestions: string[];
+  private _featuresQuestions: string[];
+  private _developerQuestions: string[];
 }
 
+class FaqWidget extends VDomWidget<FaqModel> {
+  constructor(app: JupyterLab) {
+    super();
+    this._app = app;
+    this.addClass(FAQ_CLASS);
+  }
+
+  protected render(): VNode[] {
+    let subHeadings = this.model.subHeadings;
+    let basicsQuestions = this.model.basicsQuestions;
+    let featuresQuestions = this.model.featuresQuestions;
+    let developerQuestions = this.model.developerQuestions;
+
+    // Create Frequently Asked Questions Header Section.
+    let faqHeader =
+    h.section({id: HEADER_ID},
+      h.span({className: QUESTIONMARK_ICON_CLASS + ' ' + QUESTIONMARK_CLASS}),
+      h.h1({className: HEADER_CLASS},
+        h.span({className: TITLE_CLASS},
+          this.model.title
+        )
+      )
+    );
+
+    // Create a section element that holds Table of Contents.
+    let questionList =
+    h.section({className: CONTENT_CLASS},
+      h.h2({className: SUBHEADER_CLASS}, subHeadings[0]),
+      h.ul({className: FAQ_LIST_CLASS},
+        h.li({className: QUESTION_CLASS + ' ' + TOC_CLASS},
+          h.a({href: '#basicsQ1'}, basicsQuestions[0])
+        ),
+        h.li({className: QUESTION_CLASS + ' ' + TOC_CLASS},
+          h.a({href: '#basicsQ2'}, basicsQuestions[1])
+        ),
+        h.li({className: QUESTION_CLASS + ' ' + TOC_CLASS},
+          h.a({href: '#basicsQ3'}, basicsQuestions[2])
+        ),
+        h.li({className: QUESTION_CLASS + ' ' + TOC_CLASS},
+          h.a({href: '#basicsQ4'}, basicsQuestions[3])
+        )
+      ),
+      h.h2({className: SUBHEADER_CLASS}, subHeadings[1]),
+      h.ul({className: FAQ_LIST_CLASS},
+        h.li({className: QUESTION_CLASS + ' ' + TOC_CLASS},
+          h.a({href: '#featuresQ1'}, featuresQuestions[0])
+        ),
+        h.li({className: QUESTION_CLASS + ' ' + TOC_CLASS},
+          h.a({href: '#featuresQ2'}, featuresQuestions[1])
+        )
+      ),
+      h.h2({className: SUBHEADER_CLASS}, subHeadings[2]),
+      h.ul({className: FAQ_LIST_CLASS},
+        h.li({className: QUESTION_CLASS + ' ' + TOC_CLASS},
+          h.a({href: '#developerQ1'}, developerQuestions[0])
+        ),
+        h.li({className: QUESTION_CLASS + ' ' + TOC_CLASS},
+          h.a({href: '#developerQ2'}, developerQuestions[1])
+        ),
+        h.li({className: QUESTION_CLASS + ' ' + TOC_CLASS},
+          h.a({href: '#developerQ3'}, developerQuestions[2])
+        )
+      )
+    );
+
+    // Create a section element that all other FAQ Content will go under.
+    let questionAnswerList =
+    h.section({className: CONTENT_CLASS},
+      h.h2({className: SUBHEADER_CLASS}, subHeadings[0]),
+      // Create list of questions/answers under the Basics section.
+      h.ul({className: FAQ_LIST_CLASS},
+        h.li({className: QUESTION_CLASS, id: 'basicsQ1'}, basicsQuestions[0]),
+        h.li({className: ANSWER_CLASS},
```

Comment:
@blink1073 The issue with storing answers in the model is that some answers have other DOM elements on them that makes it hard to represent simply as `strings`